src/api/getBaldiyaByCodeForWilaya.js

```diff
@@ -0,0 +1,38 @@
+const projectBaladiya = require('../utils/projections/projectObject');
+
+const getBaldiyaByCodeForWilaya = (data) =>
```

Comment:
I'd name it `getBaladyiaByWilayaCode`. But again, why we don't just use [`getWilayaByCode`](https://github.com/dzcode-io/leblad/blob/develop/src/api/getWilayaByCode.js) then call the projection.